commodore/__init__.py

```diff
@@ -2,11 +2,9 @@
 Commodore. Build dynamic inventories and compile catalogs with Kapitan
 """
 
-__author__ = 'Simon Gerber'
-__email__ = 'simon.gerber@vshn.ch'
-__license__ = 'BSD'
 __url__ = 'https://git.vshn.net/syn/commodore/'
-# NOTE: This variable is set during the Docker image build process and *MUST*
-# be set somehow during the package build process if python package builds are
-# setup in the future
-__version__ = 'Unreleased'
+__git_version__ = "0"
+
+# provide Commodore installation dir as variable that can be imported
+from pathlib import Path as P
```

Comment:
1. Imports should be placed on top of the module. Pylint would complain.
1. What is the benefit of aliasing the `Path` with `P`? This seems to obscure things. Pylint, for example, complains about single-letter variables; it will likely complain about his, too.

Reply:
Since the pylint check passes, it looks like pylint is fine with the import location and the aliasing.

`Path` is aliased here, because it's currently aliased to `P` everywhere in the Commodore code base. I don't have strong feelings on whether we should refactor the aliasing everywhere or not.